decentraland/App.js

```diff
@@ -47,9 +57,15 @@ Is the config file correct?`;
     }
   }
 
-  async _loadAccount() {
-    const account = await retrieveEphemeralAccount();
-    if (account) store.dispatch(setAccount(account));
+  async _loadAccountInfo() {
+    const account = await retrieveAccount();
+    if (account) {
+      store.dispatch(setAccount(account));
+      const creationStatus = await retrieveAccountCreationStatus();
+      const creationActions = await retrieveAccountCreationActions();
```

Comment:
I supposed maybe the function can be stringified and un-stringified?

Reply:
We’ll want to test the `getLink` function or whatever it’s called to confirm

Reply:
Since the `MyAccountScreen` isn't linking to blockchain for now, it'll work.
But I'm already aware of this issue and I'm addressing it now when I'm working on https://github.com/tasitlabs/tasit/issues/173 I'm thinking to store the `actionId` instead of `action` object.

Reply:
Okay, sounds good. And then we would do a `new action(...)` built from the actionId? Or use a separate link building function that goes straight from the id?

Reply:
As far as I've checked, change `LinkToBlockchain` component to receive an `actionId` will be enough. If the `action` object having a `buildLink` action we'll still need to resolve it and store (instead of the actionId only) because it'll be an `async` function.

Reply:
I agree that we can add another option for `LinkToBlockchain` to work that way in addition to working with a full URL.

If we started keeping the actionid (a.k.a. tx hash) as part of the action object, `buildLink` wouldn't necessarily need to be an async function.

Reply:
What do you mean by "If the `action` object having a `buildLink` action"? I had trouble parsing that part of your comment.

Reply:
Oh, I've meant an `action.buildLink` action to encapsulate the `txHash` same as `action.getId` I've thought that you made reference to this.

Reply:
Hmm, the way you're saying "`action.buildLink action`" with the word action used twice is still confusing me.

Reply:
I did make a reference to the `action.buildLink` method/function though, yep.

Reply:
> I agree that we can add another option for `LinkToBlockchain` to work that way in addition to working with a full URL.
> 
> If we started keeping the actionid (a.k.a. tx hash) as part of the action object, `buildLink` wouldn't necessarily need to be an async function.

Thoughts on this?

Reply:
Yes, I agree. I think that we could store the `actionId` from now and change to the URL in the future.

Reply:
Works for me!

Reply:
So making this change will gate merging this PR then, right?

Reply:
We can merge this PR, since the account creation links are shown only from the `BuyLandScreen`. I'm using the `actionId` on a WIP branch right now, and I think that we'll be done in a few hours.

Reply:
As long as the function on the action object can be stringified and un-stringified without creating errors - I assume you've tested this. That works for me!